orchprovider/k8s/v1/util.go

```diff
@@ -21,6 +23,79 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+type MonitoringSideCar struct {
+	// TargetIP is the IP Address of the
+	// service using which this sidecar will
+	// pull metrics info
+	TargetIP string
+
+	// Image of this sidecar
+	Image string `json:"image,omitempty"`
+
+	// SideCar is the K8s type that represents a sidecar
+	SideCar k8sApiV1.Container
+}
+
+// monSideCarTpl is the K8s type (used as a template) that gets
+// built as a sidecar
+var monSideCarTpl = k8sApiV1.Container{
+	Name:  "maya-volume-exporter",
+	Image: "openebs/m-exporter:ci",
```

Comment:
instead of hard-coding image part,can we set it as env variables or is it default value ?

Reply:
Hmmn... A lot more is there than what seems...
Can you re-read & re-review to get the answers.

This also means that this is `intelligent coding` or `smart coding` which is not at all good.
I will keep this in my mind. Thanks @utkarshmani1997

Reply:
@AmitKumarDas seems like this is the input from somewhere, are you going to pack this in openebs-operator.yaml ? 
you are also checking whether this is enabled or enabled. So where are you passing these parameters.